src/main.js

```diff
@@ -11,13 +11,14 @@ function createWindow() {
         width: 900,
         frame: false,
         transparent: true,
-        icon: "../Icons/Terminal.ico",
+        icon: path.join(__dirname, 'Icons/Terminal.ico'),
         webPreferences: {
             nodeIntegration: true,
             enableRemoteModule: true,
             experimentalFeatures: true
         }
     });
+    mainWindow.webContents.openDevTools();
```

Comment:
I think you left this here when testing lol

Reply:
fixed